### DISCUSSION TITLE
Global last update on front page?

### DISCUSSION THREAD

**Discussion by jaceklaskowski** [squidfunk/mkdocs-material#2935] [Q&A] [answered]
I use [git-revision-date](https://github.com/zhaoterryy/mkdocs-git-revision-date-plugin) for Last update that is properly displayed on every page when it changes. The "issue" is that it shows the date of the last changes to itself not some other random page.

I'd like to have this Last update on the front page (`index.md`) updated when any page changes. How?

**Comment by facelessuser** [comment, marked as answer]
Honestly, this seems like a question for: https://github.com/zhaoterryy/mkdocs-git-revision-date-plugin. Material doesn't control this functionality in any way, it just styles it. If this is not possible in the plugin, maybe you can make a request on that repository.

**Reply by jaceklaskowski** [reply]
I mis-described my wish and the plugin was merely as a reference what I'm using at the moment (and possibly not of much value when I think about it now).

I ended up with [Variables](https://squidfunk.github.io/mkdocs-material/reference/variables/) and the following on the front page (and removing the plugin whatsoever):

```
Last update: {{ git.date.strftime('%Y-%m-%d') }}
```
